projects/plugins/jetpack/tests/php/modules/masterbar/test-class-atomic-admin-menu.php

```diff
@@ -365,7 +365,8 @@ public function test_add_users_menu() {
 		$submenu = static::$submenu_data;
 
 		static::$admin_menu->add_users_menu( true );
-		$this->assertArrayNotHasKey( 2, $submenu['users.php'] );
+
+		$this->assertArrayHasKey( 2, $submenu['users.php'] );
```

Comment:
This part of the test was added only to check the exception `if ( $wp_admin ) {` exception of the `add_users_menu` function. 

We no longer have that exception, so I think we can remove this part of the test altogether since we're already checking this in line 361.

Reply:
Good point.. 8583409